lib/livebook/runtime/node_pool.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Livebook.Runtime.NodePool do
+  use GenServer, async: false
+
+  # A pool for reusing child node names.
+  #
+  # `pool` refers to the list of unused names
+  #
+  # `buffer_time` refers to time the pool waits before
+  # adding a name to `pool`,  which by default is 1 minute.
+
+  @default_time 6_000
```

Comment:
One minute would be:

```suggestion
  @default_time 60_000
```

:)